src/platform/Linux/OTAImageProcessorImpl.cpp

```diff
@@ -96,6 +127,8 @@ void OTAImageProcessorImpl::HandlePrepareDownload(intptr_t context)
         return;
     }
 
+    unlink(imageProcessor->mImageFile.data());
```

Comment:
This assumes `mImageFile` is null-terminated which doesn't have to be the case. Maybe switching from `CharSpan` to `const char*` for `mImageFile` would make sense give its use?

Reply:
Done